contracts/tasks/e2e.ts

```diff
@@ -19,69 +19,72 @@ task("e2e", "Runs all e2e tests")
     const contractAddress = taskArgs.contractAddress;
     const oracleAddress = taskArgs.oracleAddress;
 
-    await runOpenAi(
+    process.env.RUN_MODE = "e2e-script";
+
+    const testResults: Record<string, string> = {};
```

Comment:
I think it would be easier to just have a list here instead of this record?
List with some sort of type that contains info about the result, but not that important